main/reference/vstorage-ref.md

```diff
@@ -44,185 +115,193 @@ see also [Inter Protocol data](https://github.com/Agoric/agoric-sdk/tree/agoric-
     }
 ```
 
-## vstorage: agoricNames hubs
+### Namehubs - agoricNames hubs
 
 agoricNames contains several other NameHubs.
 See also [agoricNames](https://docs.agoric.com/guides/integration/name-services.html#agoricnames-agoricnamesadmin-well-known-names).
 
 ```js
-['brand', 'installation', 'instance', 'issuer', 'oracleBrand', 'vbankAsset'];
+[
+  'brand',
+  'installation',
+  'instance',
+  'issuer',
+  'oracleBrand',
+  'vbankAsset'
+];
 ```
 
-## vstorage: well known contracts
+### Well known contracts
 
 `published.agoricNames.installation` contains _Installations_ representing code of important contracts. The data at this key are the entries of the NameHub. Here we show the object comprised
 of those entries.
 See also [agoricNames in vstorage](https://docs.agoric.com/guides/integration/name-services.html#agoricnames-in-vstorage)
 regarding un-marshalling the data using board IDs.
 
 ```js
-    {
-      VaultFactory: Object @Alleged: BundleIDInstallation#board05815 {},
-      auctioneer: Object @Alleged: BundleIDInstallation#board04016 {},
-      binaryVoteCounter: Object @Alleged: BundleIDInstallation#board02314 {},
-      centralSupply: Object @Alleged: BundleIDInstallation#board0188 {},
-      committee: Object @Alleged: BundleIDInstallation#board00613 {},
-      contractGovernor: Object @Alleged: BundleIDInstallation#board02810 {},
-      econCommitteeCharter: Object @Alleged: BundleIDInstallation#board01422 {},
-      feeDistributor: Object @Alleged: BundleIDInstallation#board00917 {},
-      kreadCommitteeCharter: Object @Alleged: BundleIDInstallation#board01679 {},
-      kreadKit: Object @Alleged: BundleIDInstallation#board04980 {},
-      mintHolder: Object @Alleged: BundleIDInstallation#board02733 {},
-      priceAggregator: Object @Alleged: BundleIDInstallation#board02021 {},
-      provisionPool: Object @Alleged: BundleIDInstallation#board05311 {},
-      psm: Object @Alleged: BundleIDInstallation#board05432 {},
-      reserve: Object @Alleged: BundleIDInstallation#board00218 {},
-      scaledPriceAuthority: Object @Alleged: BundleIDInstallation#board04719 {},
-      walletFactory: Object @Alleged: BundleIDInstallation#board04312 {},
-    }
+{
+  VaultFactory: Object @Alleged: BundleIDInstallation#board05815 {},
+  auctioneer: Object @Alleged: BundleIDInstallation#board04016 {},
```

Comment:
likewise, these data were produced by a semi-automated test.

Is it important to change the indentation?

Reply:
I did it via linter. Reverting these changes.